operator/bundle.Dockerfile

```diff
@@ -1,17 +1,22 @@
 FROM scratch
 
+# Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=seldon-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=alpha
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
 LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.2.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.15.0+git
```

Comment:
is the +git expected?

Reply:
was generated but seems ok, e.g. see [here](https://github.com/open-telemetry/opentelemetry-operator/blob/7c69ad5bea42ad52846f70a5417f5c12cc6fb952/bundle.Dockerfile#L9)